src/parser.ts

```diff
@@ -90,6 +91,24 @@ interface AttributeKeyMap {
 }
 
 export class GraphMLParser {
+    public parse(text: string, cb?: Function) {
+
+        let parser = new Parser();
+
+        parser.parseString(text, (err: any, document: schema.GraphMLDocument) => {
```

Comment:
Now receives the result of parsing as a `schema.GraphMLDocument` for improving code completion.